analyzer/src/main/resources/init.gradle

```diff
@@ -120,7 +120,7 @@ class DependencyTreePlugin implements Plugin<Gradle> {
                     ResolutionResult result = configuration.getIncoming().getResolutionResult()
                     Set<ResolvedArtifact> resolvedArtifacts
                     try {
-                        resolvedArtifacts = configuration.getResolvedConfiguration().getResolvedArtifacts()
+                        resolvedArtifacts = configuration.resolvedConfiguration.lenientConfiguration.artifacts
```

Comment:
I recall we used `lenientConfiguration` already back in the old `gradle-deps-srcs/init.gradle` script. Out of curiosity, do you recall why we did not use it here from the start?